tests/sentry/api/endpoints/test_organization_join_request.py

```diff
@@ -118,4 +132,43 @@ def test_request_to_join(self, mock_record):
             "join_request.created", member_id=join_request.id, organization_id=self.organization.id
         )
 
-        assert len(mail.outbox) == 1
+        assert len(mail.outbox) == 2
+
+        assert mail.outbox[0].to == ["manager@localhost"]
+        assert mail.outbox[1].to == ["owner@localhost"]
+
+        expected_subject = f"Access request to {self.organization.name}"
+        assert mail.outbox[0].subject == expected_subject
+
+    @responses.activate
+    @with_feature("organizations:slack-requests")
+    def test_request_to_join_slack(self):
```

Comment:
For the email one I notice you check that it sends 2 (one to the manager and one to the owner), is it expected to notify both on Slack as well?

Reply:
@ceorourke yep, but I'm not checking that spec here since it's the same